.github/workflows/phpunit.yml

```diff
@@ -87,7 +87,7 @@ jobs:
       # do not stop on another job's failure
       fail-fast: false
       matrix:
-        php-versions: ['7.3', '7.4']
+        php-versions: ['7.4', '8.0']
```

Comment:
@vitormattos we need to update phpunit it seems?

Reply:
Is hardcoded on row 58, we can bump to `9.5.13`